birdhouse/birdhouse-compose.sh

```diff
@@ -13,6 +13,34 @@
 # list of all variables to be substituted in templates
 #   some of these variables *could* employ provided values in 'default.env',
 #   but they must ultimately be defined one way or another for the server to work
+
+SHELL_EXEC_FLAGS=-e
+if [ x"${BIRDHOUSE_DEBUG_MODE}" = x"true" ]; then
+  SHELL_EXEC_FLAGS=-ex
+fi
+
+set ${SHELL_EXEC_FLAGS}
+
+exit_handler() {
+  exit_code="$?"
+  [ "$exit_code" -eq 0 ] && exit
+  [ "$BIRDHOUSE_EXPECTED_EXIT" = "true" ] && exit "$exit_code"
+  err_msg="An unexpected error occurred. See additional messages on stderr for details or rerun this command with BIRDHOUSE_DEBUG_MODE=true."
+  if command -v log >/dev/null; then
```

Comment:
`command` is a built-in command?  I didn't know that !  `man command` and `info command` returns nothing !